sink/src/main/java/com/spredfast/kafka/connect/s3/sink/S3SinkTask.java

```diff
@@ -260,6 +340,7 @@ public void done() {
         }
         final BlockMetadata blockMetadata = new BlockMetadata(tp, writer.getStartOffset());
         s3.putChunk(writer.getDataFile(), writer.getIndexFile(), blockMetadata);
+        log.info("Successfully uploaded chunk for " + tp);
```

Comment:
This was added for getting the `system_test` python tests to work but I don't think it's necessary to keep as we're going to delete these tests in favor of the tests in `S3SinkConnectorIT.java`